File: Java/Logic-2/makeBricks/MakeBricks.java
package makeBricks;

public class MakeBricks {
    //Retorne true se a soma de "small" e "big" pode formar o "goal", sabendo que cada "small" vale 1 e cada "big" tem
    //valor de 5 e não é necessário usar todos para atingir a soma, apenas importa se o "goal" pode ser atingido.

    public boolean makeBricks(int small, int big, int goal) {
        int y = big*5;
        if (y > goal) y = (goal/5)*5;
        return (y + small >= goal);
    }
}

File: Python/Logic-2/make_bricks.py
"""
Sabendo que cada unidade de "small" vale 1 e que cada unidade de "big" vale 5,
retorne se é possível obter "goal" com os valores de "small" e "big" dados.
"""


def make_bricks(small, big, goal):
    big = int(goal/5) if big*5 > goal else big
    return (goal - (big*5 + small)) <= 0


print(make_bricks(2, 5, 18))
